src/groovy/grails/plugin/springsecurity/web/authentication/rememberme/GormPersistentTokenRepository.groovy

```diff
@@ -60,7 +60,10 @@ class GormPersistentTokenRepository implements PersistentTokenRepository, Grails
 		def persistentToken
 		def clazz = lookupDomainClass()
 		if (clazz) {
-			persistentToken = clazz.get(seriesId)
+			// join an existing transaction if one is active
+			clazz.withTransaction { status ->
```

Comment:
The change is consistent with the other existing methods in GormPersistentTokenRepository, all of which use withTransaction.